scripts/coding-style.sh

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Henry Schreiner
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
@SdtElectronics 
LGPL / GPL is incompatible with Apache-2.0 / Zephyr, so we can't use this script unfortunately.

I would suggest using this from upstream. It's Apache-2.0 licensed already 😁  Hopefully it works approximately the same

https://github.com/llvm/llvm-project/blob/c94a02e0e223bcce3b2c88741fae35d31a2d4f1d/clang/tools/clang-format/git-clang-format